src/development/platform-integration/ios/apple-frameworks.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Leveraging Apple's System Libraries
+description: Learn how to apply knowledge of Apple's APIs and frameworks when building Flutter apps.
+---
+
+This document is for iOS developers looking to apply
+their existing knowledge of Apple's APIs within Flutter.
+This includes accessing the device hardware and interacting 
+with specific frameworks like `HealthKit` or `MapKit`.
+
+<!-- Add once SwiftUI PR is merged -->
+<!-- For an overview of how the SwiftUI framework compares to Flutter, 
+see [Learning Flutter as a SwiftUI developer][].  -->
+
+### Flutter plugin overview
+When developing an app with Flutter, you can interact 
+with system libraries using plugins. 
+Plugins are Dart libraries that contain platform-specific code. 
+In the case of an iOS app, 
+the platform code can be written in Objective-C or Swift. 
```

Comment:
**suggestion:** Active voice. Someone performs actions in these types of docs. It doesn't happen any other way.

```suggestion
With an iOS app, you can write the code in Objective-C or Swift. 
```